cfme/cloud/provider/ec2.py

```diff
@@ -80,11 +80,16 @@ def view_value_mapping(self):
     def from_config(cls, prov_config, prov_key, appliance=None):
         """Returns the EC" object from configuration"""
         endpoint = EC2Endpoint(**prov_config['endpoints']['default'])
+        region_name = prov_config["region_name"]
+        # Note: for Version 5.10 "Northern" replace with "N." like US West (N. California)
+        if cls.appliance.version >= "5.10" and "Northern" in region_name:
```

Comment:
Don't have to include the check for `'Northern' in region_name`, because `replace` doesn't raise any exception on no match.

Reply:
we won't need an exception in conditional check.  if no match means good to go with the same name. right?

Reply:
You don't need to check that 'Northern' is in the string, before calling `string.replace('Northern', 'other')`, the call is a no-op since its not there.

The only check needed is that the appliance version is 5.10 or above, if so try to replace 'Northern' with 'N.'.